Documentation/sources/propositions/developer-documentation/classes/proposition-item.md

```diff
@@ -0,0 +1,151 @@
+# PropositionItem
+
+Represents the decision proposition item received from the remote, upon a personalization query to the Experience Edge network.
+
+```swift
+@objc(AEPPropositionItem)
+@objcMembers
+public class PropositionItem: NSObject, Codable {
+    /// Unique identifier for this `PropositionItem`
+    /// contains value for `id` in JSON
+    public let itemId: String
+
+    /// `PropositionItem` schema string
+    /// contains value for `schema` in JSON
+    public let schema: SchemaType
```

Comment:
contains value for `schema` in string, not JSON

Reply:
help me brainstorm a better way to communicate this - every schema has the following structure:
```
{
    "id": ...,
    "schema": ...,
    "data": ....
}
```

in the second line of each of these comments i'm trying to associate the variable with its corresponding json variable.

Reply:
Ah ok gotcha! What if we just add that at the top of the file instead?
```
Every PropositionItem has the following JSON format
{  
   "id": <string value>,
   "schema": <string value, must be one of known`SchemaType`,
    "data": <json object>
}
```

Reply:
For each variable in the second line we can say, `value contained by key `id`` etc. The type would already be defined in the file header